src/app/cold-storage/rebalance-to-hot-wallet.ts

```diff
@@ -0,0 +1,33 @@
+import { BTC_NETWORK } from "@config"
+import { checkedToSats, checkedToTargetConfs } from "@domain/bitcoin"
+import { TxDecoder } from "@domain/bitcoin/onchain"
+import { ColdStorageService } from "@services/cold-storage"
+import { OnChainService } from "@services/lnd/onchain-service"
+
+export const rebalanceToHotWallet = async ({
+  walletName,
+  amount,
+  targetConfirmations,
+}: RebalanceToHotWalletArgs): Promise<ColdStoragePsbt | ApplicationError> => {
+  const checkedAmount = checkedToSats(amount)
+  if (checkedAmount instanceof Error) return checkedAmount
+
+  const targetConfs = checkedToTargetConfs(targetConfirmations)
+  if (targetConfs instanceof Error) return targetConfs
+
+  const coldStorageService = await ColdStorageService()
+  if (coldStorageService instanceof Error) return coldStorageService
+
+  const onChainService = OnChainService(TxDecoder(BTC_NETWORK))
+  if (onChainService instanceof Error) return onChainService
+
+  const onChainAddress = await onChainService.createOnChainAddress()
```

Comment:
maybe I'm mistaken the use of this function, but a toHotWallet function means we need the onchain address of the lnd node (hot), not the bitcoind node (cold)

Reply:
I believe the OnChainService points to the onchain part of the lnd node (https://github.com/GaloyMoney/galoy/blob/630b1c5d78ea33384882ac1015dd95403a6883e6/src/services/lnd/onchain-service.ts#L100)